src/app/components/Headings/index.jsx

```diff
@@ -28,7 +28,10 @@ export const Headline = styled.h1`
   }
 `;
 
-export const SubHeading = styled.h2`
+export const SubHeading = styled.h2.attrs({
+  id: ({ text }) => text.replace(/\s+/g, '-'),
```

Comment:
Looks good. You'll want to strip punctuation as well, since you don't want quotes, commas, exclamation points, etc in the id.
Sorry, it seems like this was not explicitly mentioned in the issue. 
You'll want to add unit tests for this, too.

Reply:
Ok